src/antares/craft/service/api_services/output_api.py

```diff
@@ -26,11 +29,15 @@ def __init__(self, config: APIconf, study_id: str):
         self._base_url = f"{self.config.get_host()}/api/v1"
         self._wrapper = RequestWrapper(self.config.set_up_api_conf())
 
-    def read_outputs(self) -> list[Output]:
-        url = f"{self._base_url}/studies/{self.study_id}/outputs"
+    def get_matrix(self, path: str) -> pd.DataFrame:
+        return get_matrix(self._base_url, self.study_id, self._wrapper, path)
+
+    def aggregate_values(
+        self, output_id: str, aggregation_entry: AggregationEntry, mc_type: McType, object_type: ObjectType
+    ) -> pd.DataFrame:
+        url = f"{self._base_url}/studies/{self.study_id}/{object_type.value}/aggregate/{mc_type.value}/{output_id}?{aggregation_entry.to_query(object_type)}"
         try:
             response = self._wrapper.get(url)
-            outputs_json_list = response.json()
-            return [Output(name=output["name"], archived=output["archived"]) for output in outputs_json_list]
+            return pd.read_csv(StringIO(response.text))
```

Comment:
In AntaresWeb we do `pd.read_csv(io.BytesIO(response.content), sep=",")`, don't know if it's faster. We can keep the code as is but don't you have issue as you didn't specify the separator ?

Reply:
According to the doc, separator is "," by default, so no problem. As for the data handling, apparently they offer similar performance -> `StringIO, however, is a native in-memory unicode container and will exhibit similar speed to BytesIO`, from python doc

Reply:
Okay. It's weird as in pandas doc online I see , as default but in the source code it seems like it's None. Weird